scss/forms/_input-group.scss

```diff
@@ -72,7 +74,12 @@
   @include border-radius($input-border-radius-lg);
 }
 
+.input-group-lg > .form-control-plaintext{
+  padding-left: 0;
```

Comment:
Why? And why only left? I think the padding would need to be investigated some more. What are your use-cases for using form-control-plaintext inside input groups?

Reply:
Proposed this (without testing it) to handle this use case mentioned in the issue:

```html
<div class="row">
    <label for="FieldString1" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group">
            <input id="FieldString1" type="text" value="My Text" class="form-control"   />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>
<div class="row">
    <label for="FieldString2" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group">
            <input id="FieldString2" type="text" value="My Text" class="form-control" readonly />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>
<div class="row">
    <label for="FieldString3" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group">
            <input id="FieldString3" type="text" value="My Text" class="form-control-plaintext" readonly />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>

<div class="row">
    <label for="FieldString1" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group input-group-lg">
            <input id="FieldString1" type="text" value="My Text" class="form-control"   />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>
<div class="row">
    <label for="FieldString2" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group input-group-lg">
            <input id="FieldString2" type="text" value="My Text" class="form-control" readonly />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>
<div class="row">
    <label for="FieldString3" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group input-group-lg">
            <input id="FieldString3" type="text" value="My Text" class="form-control-plaintext" readonly />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>

<div class="row">
    <label for="FieldString1" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group input-group-sm">
            <input id="FieldString1" type="text" value="My Text" class="form-control"   />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>
<div class="row">
    <label for="FieldString2" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group input-group-sm">
            <input id="FieldString2" type="text" value="My Text" class="form-control" readonly />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>
<div class="row">
    <label for="FieldString3" class="col-3 col-form-label">My Label</label>
    <div class="col-9">
        <span class="input-group input-group-sm">
            <input id="FieldString3" type="text" value="My Text" class="form-control-plaintext" readonly />
            <a href="#" class="btn btn-primary">View Details</a>
        </span>
    </div>
</div>
```

Without the `padding-left: 0` the left alignment is inconsistent between medium (no left padding) and large/small versions (extra left padding).
I still need to test this PR for other use cases.

Reply:
> What are your use-cases for using form-control-plaintext inside input groups?

I reported the issue because I have a read-only input for "Company Name" and wanted a "More Info" link next to it.  

In another spot I have a read-only input for "Email Address" -- Clicking "Edit" prompts the user with "this will change your user name -- are you sure?" prompt. At which point the read-only input becomes editable.

Obviously other solution would work, but this didn't work as expected.